### ISSUE TITLE
Graphical glitch "stripes" after suspend in Hybrid mode

### ISSUE THREAD

**Issue by crispy-cat** [pop-os/pop#1383]
**Distribution (run `cat /etc/os-release`):**
```
NAME="Pop!_OS"
VERSION="20.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.10"
VERSION_ID="20.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy
LOGO=distributor-logo-pop-os
```

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
N/A, System-wide issue

**Issue/Bug Description:**
"Stripes" of the thing being rendered appear for a few frames before it completely renders like normal. I have the Gigabyte Aero 15 XB (2020 version) running in the hybrid graphics mode with Nvidia driver version 455.28. The CPU is the Intel i7-10875H and the GPU the RTX 2070 Super Max-Q. I have Firefox, Slack, Signal, xpad and a terminal window open. `nvtop` shows the only process using the Nvidia chip being `/usr/lib/xorg/Xorg`.

**Steps to reproduce (if you know):**
Put the device in suspend while in hybrid mode, then wake it up

**Expected behavior:**
Things render like normal

**Other Notes:**
None

**Comment by aspenbaily** [comment]
I'm experiencing this issue as well.

**Comment by crispy-cat** [comment]
This appears to have been fixed with the 455.38 driver update that came out today.
